### PR TITLE
chore: Replace old Xenko Game Studio images with Stride branded images

### PR DESCRIPTION
**Pull request by minktusk**
Stride Launcher section updated!

#219

### CODE REVIEW

**Comment by VaclavElias** [comment]
I should review this next week, once I am by my main dev PC.

File: en/manual/get-started/launch-stride.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ The Stride Visual Studio extension lets you you [edit shaders directly from Visu
 
 You can install multiple versions of Stride and launch them from the Stride Launcher.
 
-![Manage versions of Stride](media/stride-launcher-various-versions.png)
+![Manage versions of Stride](media/stride-launcher-various-versions.webp)
 
 You might need to use an older version of Stride to work with old projects. Newer versions of Stride might contain changes that require old projects to be upgraded.
```
Comment (VaclavElias) [inline en/manual/get-started/launch-stride.md:28]: Could this new image `media/stride-launcher-start-button.webp` replace also the one here in the line 47?

```
![Start button](media/stride-launcher-start-button.png)
```

**Comment by VaclavElias** [comment]
Thank you @minktusk. I reviewed your PR and all is ok and nice. Thank you for doing this for us. Do you want me to wait for the suggestion I made or I can merge it and my sugestion can be done at another time.

**Comment by minktusk** [comment]
Thank you @VaclavElias to review my PR! Yes, I can replace the line 47, can you wait a little more?

I have updated other sections too that I am planning to commit here!

**Comment by VaclavElias** [comment]
No problem at all. I can wait as long as you would like 🙂. No rush. Let me know once it is ready.

**Comment by minktusk** [comment]
Alright! I will send you a message when done!

**Comment by minktusk** [comment]
New better quality images and Get Started section updated!

Now ready for merge i guess!

**Comment by minktusk** [comment]
The only thing left is the Assets and Scenes section that i am planning to update in a other PR!

**Review by VaclavElias — APPROVED** [review]
Thanks all good!

**Comment by VaclavElias** [comment]
Thank you very much @minktusk. This will improve our Game Studio branding as we should see "Stride" and not Xenko.

**Pull request merged**